contributing/Readme.md

```diff
@@ -84,6 +85,34 @@ go run main.go -C hello run
 go run main.go -C hello build
 ```
 
+## Disabling Code Generation
+
+Sometimes you want to hack on the generated code in `bud/`, but if you run `bud run` any changes you make will be overriden. To avoid this, you can call your app's `main.go` file directly:
+
+```sh
+go run bud/internal/app/main.go
+```
+
+You may encounter an error like this:
+
+```sh
+budclient: discard client does not support render
+```
+
+This occurs because apps built with `bud run` depend on a bud server to provide hot reloads and client-side file bundling. To start a bud server, run the following:
```

Comment:
```suggestion
This occurs because apps built with `bud run` depends on a bud server to provide hot reloads and client-side file bundling. To start a bud server, run the following:
```